### PR TITLE
CAT-2393 Write UI tests for Chromecast

### PR DESCRIPTION
**Pull request by eisfux**
Some tests for Chromecast specific UI elements

### CODE REVIEW

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/WhenGamepadButtonBrickTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	public void setUp() throws Exception {
+		brickPosition = 1;
+		BrickTestUtils.createProjectAndGetStartScript("WhenGamepadButtonBrickTest")
+				.addBrick(new WhenGamepadButtonBrick(new WhenGamepadButtonScript()));
+		baseActivityTestRule.launchActivity(null);
+	}
+
+	@Test
+	@Flaky
+	public void whenGamepadButtonBrickTest() {
```
Comment (84n4n4) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/WhenGamepadButtonBrickTest.java]: please add 
@Category({Level.Smoke.class, Cat.AppUi.class, Cat.Gadgets.class})

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/dialog/OrientationDialogTest.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+		SharedPreferences.Editor sharedPreferencesEditor = PreferenceManager
+				.getDefaultSharedPreferences(InstrumentationRegistry.getTargetContext()).edit();
+
+		sharedPreferencesEditor.putBoolean(chromeCast, initialChromeCastSetting);
+
+		sharedPreferencesEditor.commit();
+	}
+
+	@Test
+	public void newProject() {
```
Comment (84n4n4) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/dialog/OrientationDialogTest.java]: please add
 @category({Level.Functional.class, Cat.AppUi.class, Cat.Gadgets.class})

**Comment by 84n4n4** [comment]
please just add the testing categories and levels, and the test classes to AllEspressoTestsSuite.java.

besides that, looks fine!

**Comment by cheidenreich** [comment]
I will cherry-pick your commit and make a new PR, something went wrong here

**Pull request closed without merging**